resilience4j-spring-boot2/src/main/java/io/github/resilience4j/circuitbreaker/monitoring/endpoint/CircuitBreakerServerSideEvent.java

```diff
@@ -124,4 +116,9 @@ public CircuitBreakerServerSideEvent(
     private CircuitBreaker getCircuitBreaker(String circuitBreakerName) {
         return circuitBreakerRegistry.circuitBreaker(circuitBreakerName);
     }
+
+    private Flux<ServerSentEvent<String>> getHeartbeatStream() {
+        return Flux.interval(Duration.ofSeconds(1))
```

Comment:
`getHeartbeatStream` to keep connection alive [here](https://stackoverflow.com/questions/48806452/spring-boot-webflux-netty-detect-closed-connection)